lib/inertia_rails/rspec.rb

```diff
@@ -49,14 +52,17 @@ def inertia_wrap_render(render)
       protected 
       
       def inertia_tests_setup?
-        @_inertia_render_wrapper.present?
+        ::RSpec.current_example.metadata.fetch(:inertia, false)
```

Comment:
There might be a better RSpec way to get at this data, but this seems to work ok